.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: python
 python: 2.7
-install: pip install -qq flake8 pytest pytest-cov tox
+install:
+  - pip install -qq flake8 pytest pytest-cov tox
+  - python setup.py install
```

Comment:
Because it's necessary to install dependencies (enum34). So as `enum34` is in dependencies of this package, installing package will install dependencies